Fixes #1068 - Java 11 - Build/deploy DW 4x/Accumulo 2.0.1/Hadoop 3.3.1/Wildfly 17.0.1.Final on JDK 11

keith-ratcliffe opened the pull request:
~~TODO:~~
~~1. Fix docs build~~ (fixed)
~~2. Fix ModelBeanTest~~ (fixed)

keith-ratcliffe committed:
WIP #1068 - Initial pass at building and deploying on Java 11

keith-ratcliffe committed:
Re #1068 WIP - Fix jms-client example build

keith-ratcliffe committed:
WIP #1068 - Fix quickstart dockerfile

keith-ratcliffe committed:
Temporarily skip javadoc on Github (#1068)

keith-ratcliffe committed:
WIP #1068 - No docs, temporarily

keith-ratcliffe committed:
POM cleanup (#1187)

ivakegg commented on contrib/datawave-quickstart/bin/services/datawave/test-web/tests/QueryMetrics.test:
```diff
@@ -13,7 +13,7 @@ setCurlData \
         end=20500101 \
         pagesize=1 \
-        auths=PUBLIC \
+        auths=PRIVATE \
```
This is a curious change required for Java 11... why is this needed?

keith-ratcliffe replied:
Found that the test currently fails without the PRIVATE auth, technically a separate issue. Will address in a different PR

keith-ratcliffe committed:
Re #1068:
- Fixes ModelBeanTest
- Cleans up new JAXB dependencies
- Reverts change to quickstart's QueryMetrics test

keith-ratcliffe committed:
Re #1068: Fixes javadoc issues, POM cleanup

keith-ratcliffe commented on web-services/model/src/test/java/datawave/webservice/query/model/ModelBeanTest.java:
```diff
@@ -91,8 +91,24 @@ public void setup() throws Exception {
+         *
+         * E.g., see https://stackoverflow.com/questions/52157040/
+         */
+        Class.forName("datawave.webservice.model.package-info", false, this.getClass().getClassLoader());
```
I note that other classes have the same JAXB setup but don't have this problem, such as [here](https://github.com/NationalSecurityAgency/datawave/blob/integration/warehouse/query-core/src/test/java/datawave/query/cardinality/TestCardinalityConfiguration.java#L31) and [here](https://github.com/NationalSecurityAgency/datawave/blob/integration/web-services/client/src/test/java/datawave/webservice/query/metric/QueryMetricTest.java#L149).

I also note that PowerMockRunner seems to be the main contributing factor here, given that when `@RunWith(PowerMockRunner.class)` is commented out, the JAXB deserialization works fine as-is, no workaround required

jwomeara reviewed (dismissed):
LGTM

keith-ratcliffe commented:
OBE
